packages/polka/src/index.ts

```diff
@@ -123,6 +132,12 @@ export default class Polka implements IModule, IServerHandler {
       },
     });
 
+    // Attach a traceId to every request
+    app.use((req, res, next) => {
```

Comment:
Instead of creating another middleware. We can do this inside th `handler` function.